app/src/main/res/layout/content_main.xml

```diff
@@ -6,7 +6,7 @@
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
     >
 
-    <fragment
+    <androidx.fragment.app.FragmentContainerView
```

Comment:
Could you motivate that change ?

Reply:
- It is google’s recommended view for hosting fragments.
- Fragment transactions are getting handled by FragmentContainerView under the hood.

Reply:
Ok let's try. It seems to have also some nice animation methods for fragments transition.